src/Logger/PerMutatorLogger.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Logger;
+
+use Infection\Mutant\MetricsCalculator;
+
+/**
+ * @internal
+ */
+final class PerMutatorLogger extends FileLogger
+{
+    /**
+     * @var MetricsCalculator[]
+     */
+    private $calculatorPerMutator = [];
+
+    protected function getLogLines(): array
+    {
+        $this->setUpPerCalculatorMutator();
+
+        $logs = [];
+
+        $logs[] = "# Effects per Mutator\n";
+
+        $logs[] = '| Mutator | Mutations | Killed | Escaped | Errors | Timed Out | MSI | Covered MSI |';
```

Comment:
\o 
couldcan we use https://symfony.com/doc/current/components/console/helpers/table.html helper here?

Reply:
@sidz but current PR is for markdown, not for console

Reply:
A console version of this may be a nice idea for another PR, it could be usefull to have that info printed on CI.